### PR TITLE
ruby-build: update to 20221124.

### PR DESCRIPTION
**Pull request by rhynix**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, x86_64-musl

### CODE REVIEW

**Comment by classabbyamp** [comment]
please add the email you use to commit to your github account

File: srcpkgs/ruby-build/template [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'ruby-build'
 pkgname=ruby-build
-version=20211227
+version=20221121
```
Comment (classabbyamp) [inline srcpkgs/ruby-build/template:3]: 20221123 is available

**Comment by rhynix** [comment]
> please add the email you use to commit to your github account

done

File: srcpkgs/ruby-build/template (hunk 1 above)
Comment (rhynix) [reply srcpkgs/ruby-build/template]: a new commit has been force pushed that updates the version

File: srcpkgs/ruby-build/template [hunk 2]
```diff
@@ -1,14 +1,14 @@
 # Template file for 'ruby-build'
 pkgname=ruby-build
-version=20211227
+version=20221123
```
Comment (classabbyamp) [inline srcpkgs/ruby-build/template:3]: another new version, 20221124, is available :|

Comment (rhynix) [reply srcpkgs/ruby-build/template]: The version has been updated again. ruby-build contains a lot of ruby version definitions, and every time a new definition is added or is changed, a new version is published.

**Pull request merged**